src/components/codeBlock/code-blocks.module.scss

```diff
@@ -7,6 +7,8 @@
     background: #251f3d;
     border: 1px solid #40364a;
     border-radius: 0;
+    margin-top: 0;
+    margin-bottom: 0;
```

Comment:
q: Do we have code-blocks that don't have a header? Are they still okay?

I think a less-intrusive solution is to specifically target code-blocks that have a header.